Python error with GC3Pie

Trying to build `GC3Pie` on a CentOS 7.2 host:

```
configure: creating ./config.status
config.status: creating src/config.h
warning: GMP or MPIR library not found; Not building Crypto.PublicKey._fastmath.
building 'Crypto.Hash._MD2' extension
creating build/temp.linux-x86_64-2.7
creating build/temp.linux-x86_64-2.7/src
gcc -pthread -fno-strict-aliasing -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE -fPIC -fwrapv -fPIC -std=c99 -O3 -fomit-frame-pointer -Isrc/ -I/usr/include/python2.7 -c src/MD2.c -o build/temp.linux-x86_64-2.7/src/MD2.o
src/MD2.c:31:20: fatal error: Python.h: No such file or directory
 #include "Python.h"
                    ^
compilation terminated.
error: command 'gcc' failed with exit status 1
```

--- a/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.2.3.eb
+++ b/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.2.3.eb
@@ -9,6 +9,8 @@ description = """GC3Pie is a Python package for running large job campaigns on d
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+osdependencies = [('python-devel', 'python-dev')]
+
 # this is a bundle of Python packages
 exts_defaultclass = 'PythonPackage'
 

--- a/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.3.eb
+++ b/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.3.eb
@@ -9,6 +9,8 @@ description = """GC3Pie is a Python package for running large job campaigns on d
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+osdependencies = [('python-devel', 'python-dev')]
+
 # this is a bundle of Python packages
 exts_defaultclass = 'PythonPackage'
 

--- a/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.0.eb
+++ b/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.0.eb
@@ -9,6 +9,8 @@ description = """GC3Pie is a Python package for running large job campaigns on d
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+osdependencies = [('python-devel', 'python-dev')]
+
 # this is a bundle of Python packages
 exts_defaultclass = 'PythonPackage'
 

--- a/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.1.eb
+++ b/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.1.eb
@@ -9,6 +9,8 @@ description = """GC3Pie is a Python package for running large job campaigns on d
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+osdependencies = [('python-devel', 'python-dev')]
+
 # this is a bundle of Python packages
 exts_defaultclass = 'PythonPackage'
 

--- a/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.2.eb
+++ b/easybuild/easyconfigs/g/GC3Pie/GC3Pie-2.4.2.eb
@@ -9,6 +9,8 @@ description = """GC3Pie is a Python package for running large job campaigns on d
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+osdependencies = [('python-devel', 'python-dev')]
+
 # this is a bundle of Python packages
 exts_defaultclass = 'PythonPackage'
 

